.changeset/lemon-shoes-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lion/checkbox-group': patch
+---
+
+recover possibility to registrer without []
```

Comment:
huh why? 
this enforces a best practice... we no longer want to enforce it? 🤔

Reply:
@daKmoR this apparently already didn't work. When doing the types I reenabled it, but it breaks our extension layer (so a breaking change and reverting again).

Reply:
Somehow it never hit this line before: https://github.com/ing-bank/lion/blob/master/packages/checkbox-group/src/LionCheckboxGroup.js#L16
I actually think it also conflicts with https://github.com/ing-bank/lion/blob/master/packages/form-core/src/registration/FormRegistrarMixin.js#L155

So we probably should rediscuss before Lion v1

Reply:
ok, makes sense